firefox-ios/Client/Frontend/TrackingProtection/TrackingProtectionViewController.swift

```diff
@@ -737,11 +737,15 @@ class TrackingProtectionViewController: UIViewController, Themeable, Notifiable,
 
         siteDomainLabel.text = viewModel.websiteTitle
         siteDisplayTitleLabel.text = viewModel.displayTitle
-        let totalTrackerBlocked = String(viewModel.contentBlockerStats?.total ?? 0)
-        let trackersText = String(format: .Menu.EnhancedTrackingProtection.trackersBlockedLabel, totalTrackerBlocked)
-        trackersLabel.text = trackersText
+        if let trackersNumber = viewModel.contentBlockerStats?.total, trackersNumber > 0 {
+            trackersLabel.text = String(format: .Menu.EnhancedTrackingProtection.trackersBlockedLabel,
+                                        String(trackersNumber))
+        } else {
+            trackersLabel.text = .Menu.EnhancedTrackingProtection.noTrackersLabel
+        }
```

Comment:
nit: can we separate this logic into a separate method for readability and can easily add a test

```
trackersLabel.text = getTrackerString()
```

Reply:
Tests may be added after TrackingProtectionViewController will be divided into more classes/pieces.